### PR TITLE
Modified Banner/App/Search files

### PR DESCRIPTION
**Pull request by VATSAL-IIT**
## Fixes Issue

This PR fixes the following issues:
1)Replaced deprecated hooks in Banner.js file,Search.js file and made syntactical changes to App.js file according to the latest react-version.
#example

1)In Banner.js file : Replaced useHistory with useNavigate and made changes according to useNavigate uses.
2)In app.js file replaced Switch with Routes as according to the latest version of react.Also modified code according  to latest version of react.
3)In search.js file replaced useHistory with useNavigate and made changes accordingly.
4)Replaced usage of defactored hooks according to latest react version in package.json
## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes
Screenshot (503)
Screenshot (504)](https://user-images.githubusercontent.com/78663631/152641613-7dcb4170-476e-45b6-b018-17ef17be79f0.png)
233.png)
Screenshot (506)](https://user-images.githubusercontent.com/78663631/152641692-0f574b17-0696-4d94-b03e-3d399aa0c9fc.png)
-images.githubusercontent.com/78663631/152641621-791f3d6b-e6dd-4c05-a8c3-430ff87c9ee9.png)
Screenshot (507)

### CODE REVIEW

**Comment by aman34503** [comment]
Add a screenshots please, and make sure There is no conflicts

**Comment by VATSAL-IIT** [comment]
I have added the necessary screenshots showing all the links are working properly.

**Comment by aman34503** [comment]
there is some conflicts in your code ,
try to solve this, and again open your pull request!

File: src/App.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import './App.css';
 import Home from './Home'
 import Header from './Header'
+import Banner from './Banner';
 import Footer from './Footer'
 import SearchPage from './SearchPage'
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import roundToNearestMinutes from 'date-fns/fp/roundToNearestMinutes';
```
Comment (aman34503) [inline src/App.js:9]: What does this means here?

Comment (VATSAL-IIT) [reply src/App.js:9]: Added by mistake iguess,working on it.

**Pull request closed without merging**